osu.Game/Screens/Play/HUD/HealthDisplay.cs

```diff
@@ -29,12 +32,18 @@ protected virtual void Flash(JudgementResult result)
         {
         }
 
-        [BackgroundDependencyLoader]
-        private void load()
+        [BackgroundDependencyLoader(true)]
+        private void load(HUDOverlay hud)
```

Comment:
Could annotate the param (and field, possibly) as `[CanBeNull]`.

Reply:
If we're gonna start doing this we probably want an inspection rule or something. It's only done in a handful of places we use the `CanBeNull` param.